functions/trello/conversion.ts

```diff
@@ -0,0 +1,35 @@
+import { IUnifiedIssue, Assignee } from "../common";
+
+import { List } from "./trelloTypes";
+
+export function convertTrelloDataToUnifiedIssues(
+  lists: List[],
+): IUnifiedIssue[] {
+  let unifiedIssues: IUnifiedIssue[] = [];
+
+  /*
+   * Trello doesn't provide author, statusChangeTime nor dependencies.
+   */
+  lists.forEach((list) => {
+    list.cards.forEach((card) => {
+      const unifiedIssue: IUnifiedIssue = {
+        title: card.name || "Default Title",
+        assignees: card.memberNames.map((name) => ({ name } as Assignee)),
+        author: { name: "Unknown" },
+        body: card.desc,
+        category: card.category,
+        statusChangeTime: null,
+        createdAt: card.createdDate,
```

Comment:
Will this be converted to a Date type during runtime or will it still be a string?

Reply:
Sorry, Github butchered my reference, I was asking about createdAt